contents/newsletter/first-time-founders.md

```diff
@@ -0,0 +1,133 @@
+
+> "We essentially sold BI vaporware to large companies like EMC and Cisco to help manage their business. We would use Tableau graphs embedded in wireframes to demonstrate the idea to them."
+
+They generated $500k of orders this way and worked closely with management to build the workflows. Users would email spreadsheets to an endpoint, which would then automagically populate graphs in the app. 
+
+Early feedback was glowing. It "blew the customers away". There was just one teeny, tiny problem. No one used it:
+
+> "We didn't do a good job of working with the individuals who were going to be actually using the product on a day-to-day basis. It was pretty clear that it didn't make their lives easier, and they were only using it because their boss told them to. In the end, the actual users revolted, and the product didn't give the managers enough value for them to keep making their employees use it."
+
+They persevered for a while in the hope of turning things around, but it never worked out.
+
+### What Taylor learned
+
+Initial interest and orders convinced Taylor they were onto something, but they made a few classic first-time mistakes:
+
+1. **Working on a problem they didn't know well:** While Taylor and his co-founders knew there was a problem to solve, it wasn't one they had personally encountered or tried to solve. With Mailgun, Taylor's co-founder Ev Kontsevoy "was solving his own problem that he understood deeply" and they quickly found product-market fit.
+
+2. **Talking to the wrong people:** Managers have opinions, but they rarely understand the challenges of their employees. By working with managers as design partners, rather than the people who would use the app, they created a solution that didn't survive contact with reality.
+
+3. **Building a solution that isn't 10x better or cheaper:** Points 1 and 2 inevitably lead here, as Taylor explains:
+
+> "We were very descriptive about the solution we were building and in the end it wasn't appreciably better than the email and spreadsheets based workflow they were using. People don't like change, and they won't adopt a new way of doing things unless it is massively better."
+
+## More first-time founder mistakes
+
+### Not solving a real problem
+
+Exhibit A: [Humane AI Pin](https://www.theverge.com/24126502/humane-ai-pin-review)
+
+### The wrong co-founders
+
+"[Companies are mostly about people](https://newsletter.posthog.com/p/what-we-learned-about-hiring-from)" and that starts with your co-founders. Marty Kausus quit his second startup the moment he found "the right co-founders" to start a more successful venture.
+
+They got together and "pivoted weekly for ~2 months" before landing on Pylon, a Zendesk/Intercom alternative for managing customer operations via Slack. None of his previous co-founders could commit to this level.
```

Comment:
```suggestion
They got together and "pivoted weekly" for around two months before landing on Pylon, a Zendesk or Intercom alternative for managing customer operations via Slack. None of his previous co-founders could commit to this level.
```

Reply:
we need to situate this: who is they, when did they get together, etc. also, sorry, might be me being dumb but... commit to this level of what?